cmd/plural/repos.go

```diff
@@ -54,14 +58,18 @@ func handleListRepositories(c *cli.Context) error {
 	addIcon := c.String("format") == "csv"
 
 	formatter := format.New(format.FormatType(c.String("format")))
-	header := []string{"Repo", "Description", "Publisher"}
+	header := []string{"Repo", "Description", "Publisher", "Bundle Name"}
```

Comment:
Should the header be `Bundles` or `Bundle Names` (since there are multiple)

Reply:
You beat me to it haha